src/microbe_stage/MicrobeInternalCalculations.cs

```diff
@@ -1,39 +1,147 @@
 ﻿using System.Collections.Generic;
+using System.Linq;
 using Godot;
 
 public static class MicrobeInternalCalculations
 {
+    public static Vector3 MaximumSpeedDirection(IEnumerable<OrganelleTemplate> inputOrganelles)
```

Comment:
Why is this called `inputOrganelles`I think the parameter should just be called `organelles` so that it doesn't confuse the future code writer as to why this takes in "inputOrganelles" and not just "organelles"?
You can rename the local variable to movementOrganelles.